test/index-test.js

```diff
@@ -1,45 +1,49 @@
-const chai = require('chai')
-const fs = require('fs')
-const jsdom = require('mocha-jsdom')
-const path = require('path')
-const spies = require('chai-spies')
+const chai = require('chai');
+const fs = require('fs');
+const jsdom = require('mocha-jsdom');
+const path = require('path');
+const spies = require('chai-spies');
 
-chai.use(spies)
+chai.use(spies);
 
-const expect = chai.expect
+const expect = chai.expect;
 
 
-describe('index', () => {
+describe('index.js', () => {
   jsdom({
     src: fs.readFileSync(path.resolve(__dirname, '..', 'index.js'), 'utf-8')
-  })
+  });
 
-  describe('changeCompletely(element, index, array)', () => {
+  describe('doToElementsInArray()', () => {
     it('is defined', () => {
-      expect(changeCompletely).to.be.a('function')
-    })
+      expect(doToElementsInArray).to.be.a('function');
+    });
 
-    it('completely alters an array in place when used with Array.prototype.forEach', () => {
-      const array = [1, 2, 3]
+    it('invokes the passed-in callback function on every element of the passed-in array using Array.prototype.forEach()', () => {
+      const callback = fruit => `Mmmm, ${fruit}!!!`;
```

Comment:
I hear you, but the lesson is on `Array.prototype.forEach()`, which is explicitly for operating on an array. The preceding lesson is all about callbacks. Or am I completely missing the point you're making? 😅